WNPRC_Purchasing/package.json

```diff
@@ -21,7 +21,7 @@
     "@fortawesome/free-solid-svg-icons": "^6.4.2",
     "@fortawesome/react-fontawesome": "^0.2.0",
     "@labkey/api": "1.27.0",
-    "@labkey/components": "2.390.4",
+    "@labkey/components": "2.390.6",
```

Comment:
10k lines changes in the package-lock.json seems like a lot for a patch version bump of the @labkey/components package. That may very well be the case if this package.json wasn't updated in awhile, but just calling it out as something to double check.

Reply:
good point, reverted this change in both package.json and package-lock.json